<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fhc</class>
 <widget class="QWidget" name="fhc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PrimaryPushButton" name="btn_A">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="PrimaryPushButton" name="btn_B">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="PlainTextEdit" name="lineedit_A">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PlainTextEdit" name="lineedit_B">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="TableWidget" name="tablewidget">
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>字符数</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>文件大小</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>CRC32</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>md5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>sha1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>sha224</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>sha256</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>sha384</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>sha512</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>A区哈希值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>比较结果</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B区哈希值</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_A</tabstop>
  <tabstop>lineedit_A</tabstop>
  <tabstop>btn_B</tabstop>
  <tabstop>lineedit_B</tabstop>
  <tabstop>tablewidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
